common/strings/split.h

```diff
@@ -54,7 +54,7 @@ class StringSpliterator {
   StringSpliterator& operator=(StringSpliterator&&) = default;
 
   // Returns true if there is at least one result to come.
-  operator bool() const { return !end_; }
+  operator bool() const { return !end_; }  // NOLINT
```

Comment:
The bool conversion is intended to be able to use in a while loop more conveniently.
Can NOLINT be annotated with the exact rule being waived, for clarity?

Reply:
Done (rule is `google-explicit-constructor`)